docs/proposals/20210310-opt-in-autoscaling-from-zero.md

```diff
@@ -188,6 +193,29 @@ type DockerMachineTemplate struct {
 ```
 _Note: the `ResourceList` and `ResourceName` referenced are from k8s.io/api/core/v1`_
 
+`platform.NodeSystemInfo` is a struct that contains the architecture and operating system information of the node, to 
+implement in the `util` package of the `cluster-api` project. The defined types and constants are exported for use by the 
+cluster-api providers integrations.
+Its definition would look like this:
+
+```go
+package platform
+
+type Architecture string
```

Comment:
Is the idea to make this an enum? (i.e. enumerate all valid values)

If yes, let's add the appropriate enum marker here